site/formulas/decoding.md

```diff
@@ -7,14 +7,14 @@ follows.
 1. Check that `s_bytes` is the canonical encoding of a field element, or else abort.
 2. Decode `s_bytes`, a byte-string, into \\(s\\), a field element.
 3. Check that \\(s\\) is non-negative, or else abort.
-4. \\(u_1 \gets 1 - a s^{2} \\).
-5. \\(u_2 \gets 1 + a s^{2} \\).
+4. \\(u_1 \gets 1 + a s^{2} \\).
+5. \\(u_2 \gets 1 - a s^{2} \\).
 6. \\(v \gets adu_1^2 - u_2^2 \\).
-7. \\(I \gets \mathrm{invsqrt}( v u_1^{2} ) \\), or abort if the square root does not exist.
+7. \\(I \gets \mathrm{invsqrt}( v u_2^{2} ) \\), or abort if the square root does not exist.
```

Comment:
I see `I = 1/sqrt(v u_1)` in the [code](https://github.com/dalek-cryptography/curve25519-dalek/blob/39145da396226516d8d1f663ad26f86bf04856db/src/ristretto.rs#L268)?